lib/webapp.js

```diff
@@ -172,6 +172,108 @@ const staticHandlers = {
     });
     return r;
   },
+
+  server(options){
+    const router = express.Router();
+    const dataserviceAuth = options.serverConfig["dataserviceAuthentication"];
+    const rbac = (dataserviceAuth != undefined) ? dataserviceAuth.hasOwnProperty("rbac"): false;
+    if(!rbac || rbac && dataserviceAuth.rbac == false){
+      router.get('/*', (req, res) => {
+        res.status(401).send("Unauthorized - RBAC is disabled.  Enable in zluxserver.json");
```

Comment:
Change to 400.  Not authorized, server not configured correctly.  Change message to "Set dataserviceAuthentication.rbac to true in server configuration"